tutor/test_cc1_03_ContentPreparationAndImport.py

```diff
@@ -37,14 +38,14 @@ class TestContentPreparationAndImport(unittest.TestCase):
 
     def setUp(self):
         """Pretest settings."""
-        raise NotImplementedError(inspect.currentframe().f_code.co_name)
-
         self.ps = PastaSauce()
         self.desired_capabilities['name'] = self.id()
         self.content = ContentQA(
             use_env_vars=True,
             pasta_user=self.ps,
-            capabilities=self.desired_capabilities
+            capabilities=self.desired_capabilities,
+            username=os.getenv('CONTENT_USER'),
```

Comment:
I was being stupid and commented out use_env_vars=True. Just tested it there's no issue with this